Dockerfile

```diff
@@ -1,9 +1,10 @@
-FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.11 AS builder
-WORKDIR /go/src/github.com/nutanix-cloud-native/machine-api-provider-nutanix
+
```

Comment:
Nit should remove this empty line

Reply:
Still seems to be there 😅  Also, please make sure this file ends with a new line, that's missing at the moment